frontend/src/components/Psychologist/Patients/patientRow.cssmodule.scss

```diff
@@ -0,0 +1,7 @@
+.incomplete {
+    background-color: var(--orange-soft-400);
+}
+
+.complete {
+    background-color: var(--green-soft-400);
+}
```

Comment:
* I would personally use `300` instead of `400`. It's really flashy ATM.
* On mobile, there is still no color in the icon.